### PR TITLE
feat: honest form in blog

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/blocks/Form/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import './Form.scss';
+
+const b = block('form-block');
+
+export const Form = ({paddingTop, paddingBottom, ...props}: FormProps) => {
+    const {formData, border = 'shadow'} = props;
+    const [contentLoaded, setContentLoaded] = useState(false);
+
+    const onContentLoad = useCallback(() => {
```
Comment (NikitaCG) [inline src/blocks/Form/Form.tsx:18]: I think `useCallback` in this place is not necessary

File: src/blocks/Form/__stories__/Form.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    title: 'Blocks/Form',
+    component: Form,
+    args: {
+        theme: 'light',
+    },
+} as Meta;
+
+const __getFormData = (formData: FormBlockModel['formData']) => {
+    const id = uuidv4();
+    console.log({formData});
```
Comment (NikitaCG) [inline src/blocks/Form/__stories__/Form.stories.tsx:25]: delete this, if we dont need it

File: .mocks/utils.ts [hunk 3]
```diff
@@ -61,5 +63,6 @@ export const getVideoStoryArgs = () => {
     };
 };
 
+export const getFormsData = () => forms;
```
Comment (NikitaCG) [inline .mocks/utils.ts:66]: why we dont export constant ?

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/blocks/Form/Form.tsx (hunk 1 above)
Comment (niktverd) [reply src/blocks/Form/Form.tsx]: yeap

File: src/blocks/Form/__stories__/Form.stories.tsx (hunk 2 above)
Comment (niktverd) [reply src/blocks/Form/__stories__/Form.stories.tsx]: removed

File: .mocks/utils.ts (hunk 3 above)
Comment (niktverd) [reply .mocks/utils.ts]: changed

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**